docs/ProjectSpec.md

```diff
@@ -218,6 +218,12 @@ These only applied to `target` and `framework` dependencies.
 - ⚪️ **codeSign**: `Bool` - Whether the `codeSignOnCopy` setting is applied when embedding framework. Defaults to true
 - ⚪️ **removeHeaders**: `Bool` - Whether the `removeHeadersOnCopy` setting is applied when embedding the framework. Defaults to true
 
+**Carthage link option for macOS**:
+
+For macOS, Carthage frameworks are to be always embedded. But sometime you need to link, too.
```

Comment:
I'm not sure about this. As far as I know, the frameworks generated by Carthage needs to be linked and embedded regardless of the platform.

Reply:
You are right. That's safer.🤔
Do you want me to fix the embed option to defaults to true for macOS?

Reply:
I think that would prevent that developers generate the project and it doesn't run because frameworks are missing. I don't see a case where you would link the Carthage frameworks without embedding them.

Reply:
Well it's working for me, though?
Anyways it's safer to both link and embed, so I'm closing this PR.👌
I will submit another fix for you, because it was only linking Carthage frameworks when I tried.

Reply:
What do you think @yonaskolb ?